examples/README.md

```diff
@@ -106,146 +109,139 @@ the wrong directory.
-This is because I don't have default mapped to 192.168.242.55 in my hosts file.  Rather than mess with that.  Lets just
-take the port information and add it to the IP of where we know the vagrant "cluster" is located.   Open the browser to
- 192.168.242.55:9061 (in this example) and we see:
+If you click on the `default:9061` it will open your browser to default:9061 and
+get a `webpage is not avaiable` error. This is because you don't have default
+mapped to the Vagrant VM's IP address in my hosts file.  Rather than mess with
+that, you can just take the port information and add it to the IP of where we
+know the Vagrant machine is located. Open the browser to `<VM-IP>:9061` and
+you'll see:
 
  ![hello world application](./images/helloworld.png)
 
-
-Note that it has the container # in it.  If you go back and look at marathon and get the other Port. This will take you
-to the other container.  if you look into the json file for [hello world](hello-world/hello-world.json) you will see that
-you are submiting a call to create two instances of a docker image. `"image": "keithchambers/docker-hello-world",`.  Google that
-and you get to the [php code running in this container](https://github.com/keithchambers/docker-hello-world/blob/master/index.php).
+Note that it has the container number in it. If you go back and look at
+Marathon, you can navigate to the other port and find the other container.
+If you look into the json file for [hello world](hello-world/hello-world.json)
+you will see that you are submiting a call to create two instances of a docker
+image: `"image": "keithchambers/docker-hello-world"`. You can see the the [php code running in this container](https://github.com/keithchambers/docker-hello-world/blob/master/index.php).
 
 ## Destory the Vagrant Cluster and Build on for MineCraft
 
-If you want to use the vagrant cluster for the Minecraft example you have to make it a bigger VM.  Minecraft requires
-2 Gigs free for itself.
+If you want to use the vagrant cluster for the Minecraft example you have to
+make a worker VM with more resources. Minecraft requires 2GB of RAM free for
+good performance.
 
-Easy enough!
+Easy enough! If you have the resources on your computer, you can
 
 1. Destroy your vagrant server. Go to the project root and type:
 
         vagrant destroy
 
-2. Edit the vagrant file.
-
-Change `Vagrantfile` lines near the bottom from
-
-          config.vm.provider :virtualbox do |vb|
-            vb.customize ['modifyvm', :id, '--cpus', 1]
-            vb.customize ['modifyvm', :id, '--memory', 1536]
-          end
-
-to
+2. Create a `vagrant-config.yml` file in the root directory, and add
 
-          config.vm.provider :virtualbox do |vb|
-            vb.customize ['modifyvm', :id, '--cpus', 2]
-            vb.customize ['modifyvm', :id, '--memory', 3536]
-          end
+```
+---
+worker_memory: 3072
```

Comment:
with #974, this will need to be adjusted to ensure you also include `/vagrant/terraform.sample.yml` in the playbooks variable in `vagrant-config.yml`